lib/sys.py

```diff
@@ -17,13 +17,15 @@
 from __go__.os import Args, Stdin, Stdout, Stderr
 from __go__.grumpy import SysModules, MaxInt, NewFileFromFD  # pylint: disable=g-multiple-import
 from __go__.runtime import Version
+from __go__.math import MaxInt64
 
 argv = []
 for arg in Args:
   argv.append(arg)
 
 goversion = Version()
 maxint = MaxInt
+maxsize = MaxInt64 - 1
```

Comment:
I think generally we use grumpy.MaxInt as a max size for sequences in Grumpy. So yeah I think you can just make maxint and maxsize the same.